src/XenobladeChroniclesX/Mods/WeatherForceWeather/rules.txt

```diff
@@ -117,7 +117,7 @@ category = Weather
 condition = $region == 3
 
 [Preset]
-name = "Rain [2]"
+name = "Rain [2] "
```

Comment:
```suggestion
name = "Rain [2]"
```

Reply:
the space is a bug fix for a naming conflict, it needs to be added.
back in 2023 I had someone @me about this bug in particular